test/skimNtuple_HHbtag.cpp

```diff
@@ -3202,35 +3204,36 @@ int main (int argc, char** argv)
 				  double Eff_Data = passSingle * SFL_Data - passCross * passSingle * std::min(SFl_Data, SFL_Data) * SFtau_Data + passCross * SFl_Data * SFtau_Data;
 				  double Eff_MC   = passSingle * SFL_MC   - passCross * passSingle * std::min(SFl_MC  , SFL_MC)   * SFtau_MC   + passCross * SFl_MC   * SFtau_MC;
 
-				  double Eff_Data_mu_up   = passSingle * SFL_Data_up   - passCross * passSingle * std::min(SFl_Data_up  , SFL_Data_up)   * SFtau_Data + passCross * SFl_Data_up   * SFtau_Data;
-				  double Eff_MC_mu_up     = passSingle * SFL_MC_up     - passCross * passSingle * std::min(SFl_MC_up    , SFL_MC_up)     * SFtau_MC   + passCross * SFl_MC_up     * SFtau_MC;
-				  double Eff_Data_mu_down = passSingle * SFL_Data_down - passCross * passSingle * std::min(SFl_Data_down, SFL_Data_down) * SFtau_Data + passCross * SFl_Data_down * SFtau_Data;
-				  double Eff_MC_mu_down   = passSingle * SFL_MC_down   - passCross * passSingle * std::min(SFl_MC_down  , SFL_MC_down)   * SFtau_MC   + passCross * SFl_MC_down   * SFtau_MC;
-
-				  // for each DM, fill a trigSF branch with the up/down values if the tauh has the corresponding DM, otherwise fill with nominal trigSF value
-				  vector <double> SFtau_Data_up   (N_tauhDM, SFtau_Data);
-				  vector <double> SFtau_Data_down (N_tauhDM, SFtau_Data);
-				  vector <double> SFtau_MC_up     (N_tauhDM, SFtau_MC);
-				  vector <double> SFtau_MC_down   (N_tauhDM, SFtau_MC);
-				  vector <double> Eff_Data_up     (N_tauhDM, Eff_Data);
-				  vector <double> Eff_Data_down   (N_tauhDM, Eff_Data);
-				  vector <double> Eff_MC_up       (N_tauhDM, Eff_MC);
-				  vector <double> Eff_MC_down     (N_tauhDM, Eff_MC);
-
-				  for (int idm  = 0; idm < N_tauhDM; idm ++)
-					{
-					  if (isthisDM_second[idm])
-						{
-						  SFtau_Data_up[idm]   = tauTrgSF_mutau->getEfficiencyData(tlv_secondLepton.Pt(), DM2, 1);
-						  SFtau_Data_down[idm] = tauTrgSF_mutau->getEfficiencyData(tlv_secondLepton.Pt(), DM2, -1);
-						  SFtau_MC_up[idm]     = tauTrgSF_mutau->getEfficiencyMC  (tlv_secondLepton.Pt(), DM2, 1);
-						  SFtau_MC_down[idm]   = tauTrgSF_mutau->getEfficiencyMC  (tlv_secondLepton.Pt(), DM2, -1);
+				  // Compute uncertainties (separate for SingleMuon and mu leg of cross trigger)
+				  double Err_Data_SL_mu = passSingle * SFL_Data_Err - passCross * passSingle * (SFl_Data > SFL_Data) * SFL_Data_Err * SFtau_Data;
```

Comment:
GetErrorYhigh() will return whatever was stored as the error. Previously the TGraphAsymmErrors were filled in a way that the error would correspond to nominal+uncertainty (or nominal-uncertainty for the ErrorYlow), but I changed that. So now GetErrorYhigh() will just return the uncertainty (same as GetErrorYlow() since they are symmetric in our case)

Reply:
Ok, thanks. This was not clear from the documentation. Could you please add a comment repeatng what you say above next to ```GetErrorYhigh()```, to avoid future confusion?